docs/components/modeler/web-modeler/additional-features/call-activity-linking.md

```diff
@@ -0,0 +1,27 @@
+---
+id: call-activity-linking
+title: Call activity linking
+description: Call activity linking
+---
+
+You can use one of the following approaches to link a [call activity](/components/modeler/bpmn/call-activities/call-activities.md) to another process.
+
+### Using the link button
+
+If you select a call activity, a link button appears at the bottom right. Clicking on the button opens a dialog from which you can choose any diagram from the same project. Click on the `Link` button after selecting the diagram. This will create the link by copying the process ID of the diagram to the `Called element` section in the properties panel.
+
+![overlay](img/overlay.png)
```

Comment:
Can we use better examples in the screenshot (Something real-world like "Invoke payment" or similar, especially having both callee and caller here is confusing, would just expect the callee to appear-I know self-calling is possible but it is still not the best case to show)

Reply:
Best would be to use names of processes that re-appear in the docs, so we have one consistent "storyline" throughout the docs